### PR TITLE
feat(ci): add `lightwalletd_update_sync` test to CI

### LINKED ISSUE #4167: ci: call lightwalletd quick tip sync test
## Motivation

The test from #4166 must be invoked in CI using the ligthwalletd cached state (#4165)

### PR DESCRIPTION
**Pull request by gustavovalverde**
## Motivation

We want to add all completed `lightwalletd` integration tests to CI, to confirm we're not introducing bugs to the RPC endpoints or any other implementation related to `lightwalletd`

Fixes #4167
Depends-On: #4268

### Designs

- Consider the implementations being done in `test.yml` and `test-full-sync.yml`

## Solution

- Add `lightwalletd_update_sync` test to CI
- Add `lightwalletd_update_sync` to `entrypoint.sh` for easier execution

## Review

Anyone from the DevOps team can review this

### Reviewer Checklist

  - [ ] Verify tests are using the right execution methods
  - [ ] Confirm job id, step names and description

## Follow Up Work

Simplify this job with the refactor being done for reusable workflows

### CODE REVIEW

**Comment by teor2345** [comment]
Not required to merge PR #4267, which is our top priority right now.

File: .github/workflows/deploy-gcp-tests.yml
Comment (teor2345) [inline .github/workflows/deploy-gcp-tests.yml]: We repeat the disk prefix calculation twice, but the result is exactly the same both times.
Can you do it before the queries, and put the result in an environmental variable?

That will simplify the code a bit.

File: .github/workflows/deploy-gcp-tests.yml
Comment (teor2345) [inline .github/workflows/deploy-gcp-tests.yml]:
```suggestion
      # - disk cached state (lwd_state_dir/zebra_state_dir or disk_prefix) - zebrad-cache or lwd-cache
```

File: .github/workflows/deploy-gcp-tests.yml
Comment (gustavovalverde) [reply .github/workflows/deploy-gcp-tests.yml]: I'm not sure I understood your suggestion. You can push to this branch if you'd like, and apply the changes directly.

**Comment by gustavovalverde** [comment]
This is up to date with #4268, and it the test has been working so far.

**Review by dconnolly — DISMISSED** [review]
lgtm! 📦📤📥🔁

**Comment by gustavovalverde** [comment]
The CI is working as expected now, this is failing as the `lightalletd full sync` is also failing due to a bug in `lightwalletd`, documented here https://github.com/ZcashFoundation/zebra/blob/main/.github/workflows/zcash-lightwalletd.yml#L55-L60

While the `lightalletd full sync` keeps failing, we won't have a disk with the new naming requirements, making the `lightalletd update sync` to also fail.

But that issue is not related to this PR, and should be solved in https://github.com/adityapk00/lightwalletd

**Comment by dconnolly** [comment]
> The CI is working as expected now, this is failing as the `lightalletd full sync` is also failing due to a bug in `lightwalletd`, documented here https://github.com/ZcashFoundation/zebra/blob/main/.github/workflows/zcash-lightwalletd.yml#L55-L60
> 
> While the `lightalletd full sync` keeps failing, we won't have a disk with the new naming requirements, making the `lightalletd update sync` to also fail.
> 
> But that issue is not related to this PR, and should be solved in https://github.com/adityapk00/lightwalletd

So this is blocked by the lightwalletd changes?

**Comment by gustavovalverde** [comment]
> So this is blocked by the lightwalletd changes?

Considering the issue has been impacting `lightwalletd_full_sync` since we started working on it, and as a consequence it also has an impact on this one, but we agreed on merging `lightwalletd_full_sync` anyways, I'm not sure if we should block this one until a fix hits `upstream` lightwalletd. 

But that's not my call.

**Comment by gustavovalverde** [comment]
I'll at least fix the images name so we can ensure `lightwalletd_update_sync` is working before and after merging.

**Comment by gustavovalverde** [comment]
@dconnolly we got a successful run here. This one is ready from my side https://github.com/ZcashFoundation/zebra/runs/6430109707?check_suite_focus=true

After merging I'll take on https://github.com/ZcashFoundation/zebra/pull/4303 so we can close this initial lightwalletd cycle

**Review by dconnolly — APPROVED** [review]
🎉

**Pull request merged**
